.github/scripts/rust/Makefile.toml

```diff
@@ -0,0 +1,157 @@
+[config.modify_core_tasks]
+private = true
+namespace = "default"
+
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+CARGO_HACK_FLAGS = "--feature-powerset --exclude-features default"
```

Comment:
Does this need more flags from the previous CI? 

The complete flag list used to be `--feature-powerset --features build-nng --exclude-features default --ignore-unknown-features --optional-deps clap`, is there more of that that is not engine specific?